awx/main/analytics/collectors.py

```diff
@@ -0,0 +1,236 @@
+        'execution_node', 'status').annotate(job_status=Count('status'))
+    for job in job_statuses:
+        counts.setdefault(job[0], {}).setdefault('launch_type', {})[job[1]] = job[2]
+    return counts
+
+
+# Copies Job Events from db to a .csv to be shipped
+def copy_tables(since, full_path):
+    def _copy_table(table, query, path):
+        events_file = os.path.join(path, table + '_table.csv')
+        try:
+            write_data = open(events_file, 'w', encoding='utf-8')
+            with connection.cursor() as cursor:
+                cursor.copy_expert(query, write_data)
+                write_data.close()
+        except Exception as e:
+            return e
+        return events_file
+
+    events_query = '''COPY (SELECT main_jobevent.id, 
+                              main_jobevent.created, 
+                              main_jobevent.event, 
+                              main_jobevent.failed, 
+                              main_jobevent.changed, 
+                              main_jobevent.playbook, 
+                              main_jobevent.role, 
+                              main_jobevent.job_id, 
+                              main_jobevent.host_id, 
+                              main_jobevent.host_name, 
+                              FROM main_jobevent 
+                              WHERE main_jobevent.created > {}
+                              ORDER BY main_jobevent.id ASC) to stdout'''.format(since.strftime("'%Y-%m-%d %H:%M:%S'"))
+    _copy_table(table='events', query=events_query, path=full_path)
+    
+    unified_job_query = '''COPY (SELECT DISTINCT main_unifiedjob.id, 
+                                 main_unifiedjob.polymorphic_ctype_id,
+                                 django_content_type.model,
+                                 main_unifiedjob.created, 
+                                 main_unifiedjob.description, 
+                                 main_unifiedjob.created_by_id, 
+                                 main_unifiedjob.name,  
+                                 main_unifiedjob.unified_job_template_id, 
+                                 main_unifiedjob.launch_type, 
+                                 main_unifiedjob.schedule_id, 
+                                 main_unifiedjob.execution_node, 
+                                 main_unifiedjob.controller_node, 
+                                 main_unifiedjob.cancel_flag, 
+                                 main_unifiedjob.status, 
+                                 main_unifiedjob.failed, 
+                                 main_unifiedjob.started, 
+                                 main_unifiedjob.finished, 
+                                 main_unifiedjob.elapsed, 
+                                 main_unifiedjob.job_explanation, 
+                                 main_unifiedjob.start_args, 
+                                 main_unifiedjob.result_traceback, 
+                                 main_unifiedjob.celery_task_id, 
```

Comment:
I think we should lose:

``
`start_args` (has had sensitive data leaks, historically)
`celery_task_id` (useless info)